src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -237,7 +238,7 @@ protected function _setupTable($fixture, $db, array $sources, $drop = true)
         $exists = in_array($table, $sources);
 
         if (($drop && $exists) ||
-            ($exists && !$isFixtureSetup && $fixture->schema() instanceof Table)
+            ($exists && !$isFixtureSetup && $fixture instanceof TableSchemaInterface && $fixture->schema() instanceof Table)
```

Comment:
We've been following the same pattern in other places... I personally prefer having an interface to describe what the method is about. Specially becuase you could have another interface in another class with the same method name but with a different behavior.

If you think this is not wroth it, we can of course just do method_exists(), I'm not attached to this solution